base/applications/mspaint/dialogs.cpp

```diff
@@ -159,6 +165,21 @@ LRESULT CAttributesDialog::OnRadioButton3(WORD wNotifyCode, WORD wID, HWND hWndC
     return 0;
 }
 
+LRESULT CAttributesDialog::OnRadioButton4(WORD wNotifyCode, WORD wID, HWND hWndCtl, BOOL& bHandled)
```

Comment:
I know the original code use these names, but I think you could use better self-documenting names for these new functions.
What is RB4 and 5?

Reply:
I agree with your suggestion but there are also other names not self-documenting so I decided to use the originals. 
I could refactor them in next time.
RB4 is the radiobutton related to switch on "black and white mode", while RB5 is the radiobutton related to switch on "colors mode" inside Attributes Dialog, as you can see in this pictures:

![image](https://user-images.githubusercontent.com/55651510/114321440-14c7e900-9b1b-11eb-9155-c036fa6e1d4d.png)